### PR TITLE
Add April 2024 newsletter to the website

### LINKED ISSUE #1148: Add April 2024 newsletter to the website
## Description

Every month, we try to get the newsletter up on the site within a week of emailing it. Currently, we're moving them over "by hand."

## Steps to update

In the code base, navigate to `app > routes > newsletter > issues` and create a new file `2024-04.jsx`.
You can look at the existing newsletters ( `app > routes > newsletter > issues`) as a template.

Make sure to add it to the index by following the steps in the ["Newsletters" section in our README](https://github.com/Virtual-Coffee/virtualcoffee.io#newsletters) and update the content accordingly based on our email newsletter.

### PR DESCRIPTION
**Pull request by adiati98**
## Linked Issue

Closes #1148 

## Description

This PR adds April 2024 newsletter to the website and updates the `newsletters.ts`.

### Live Preview

https://deploy-preview-1149--virtual-coffee-io.netlify.app/newsletter/issues/2024-04

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+			<hr />
+
+			<h2 className="my-5">📆 What's happening at Virtual Coffee</h2>
+			<h3>March Recap</h3>
+			<p>
+				<strong>💡Monthly Theme & Challenge: Getting Job Ready!</strong>
+			</p>
+			<p>
+				We spent the month working on resumes, cover letters, interview skills
+				and all thing getting ready to work! Shoutout to all our community
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:69]: **nitpick:** Add the Oxford comma to keep it consistent throughout

```suggestion
				We spent the month working on resumes, cover letters, interview skills,
				and all thing getting ready to work! Shoutout to all our community
```

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+				members who participated!
+			</p>
+
+			<h3 className="mt-4">April: Find Your Voice with Public Speaking!</h3>
+			<p>
+				<strong>💡Monthly Theme & Challenge: Public Speaking!</strong>
+			</p>
+			<p>
+				Whether you're preparing for a keynote, a tech talk, or just want to get
+				more comfortable speaking in public, we've got your back.
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:79]: **praise:** I love this line! ❤️

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+				<Link to="/monthlychallenges/apr-2024">April 2024 challenge page</Link>{' '}
+				on the Virtual Coffee website.
+			</p>
+
+			<h3 className="mt-4">☕🪑 Coffee Table Events</h3>
+			<ul>
+				<li>Tech Interview Study Group (Mondays at 4:00 PM ET)</li>
+				<li>
+					Accountabilibuddies (Tuesdays at 7:00 PM ET and Thursdays at 9:00 AM
+					ET)
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:98]: **nitpick:** Another consistency thing. Down below you have `Virtual Coffee (Tuesdays at 9:00 AM ET | Thursdays at 12:00 PM ET)` and here you use `Accountabilibuddies (Tuesdays at 7:00 PM ET and Thursdays at 9:00 AM ET)`. Should we use `|` or `and`?

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+				</li>
+				<li>The Pack Hunt (Tandem Job Hunting) (Wednesdays at 2:00 PM ET)</li>
+				<li>Feelings Friday (Fridays at 10:00 AM ET)</li>
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:101]: **issue:** Feelings Friday is at 8:00 PM ET on Fridays

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+				</li>
+				<li>
+					Tuesdays: Trivia in <code>#game-night</code>
+				</li>
+				<li>
+					Wednesdays: Midweek check-in in <code>#general</code>
+				</li>
+				<li>
+					Thursdays: New book club questions for Cracking the Coding Interview
+					on GitHub
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:120]: **issue:** This has fallen off. I've spoken with Lauren and I know she's got a lot going on so she may have decided to take a hiatus.

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+				</li>
+				<li>
+					Fridays:{' '}
+					<a href="https://twitter.com/VirtualCoffeeIO">X (formerly Twitter)</a>{' '}
+					chat and gratitude post in <code>#general</code>
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:125]: **question:** Can we add the Rust Learning Cohort check-in in #learning-together here?

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 7]
```diff
@@ -0,0 +1,370 @@
...
+				</li>
+			</ul>
+
+			<h3 className="mt-4">☕ Official Virtual Coffee Events</h3>
+			<ul>
+				<li>
+					Virtual Coffee (Tuesdays at 9:00 AM ET | Thursdays at 12:00 PM ET)
+				</li>
+				<li>
+					Async Twitter Chat (Fridays — prompts tweeted out at 9:00 AM ET)
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:135]: **suggestion (non-blocking)** Maybe something like...

```suggestion
					Async Twitter Chat (Weekly chat begins every Friday at 9:00 AM ET)
```

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 8]
```diff
@@ -0,0 +1,370 @@
...
+			</p>
+
+			<h3>🆕 New Career Focus Channels Alert</h3>
+			<p>
+				We've added three new private channels — <code>Senior+</code>,{' '}
+				<code>Mid-level</code>, <code>Early-career</code> — to our Slack for
+				peer-to-peer conversation, because we know that it's good to be able to
+				support and nurture conversations specific to your career level. If
+				you're interested in joining the one you identify with most, DM Bekah on
+				Slack.
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:154]: **issue:** Add me in there too so it's not all on Bekah

```suggestion
				support and nurture conversations specific to your career level. If
				you're interested in joining the one you identify with most, DM Bekah or
				Meg on Slack.
```

File: app/routes/__frontend/newsletter/issues/2024-04.jsx [hunk 9]
```diff
@@ -0,0 +1,370 @@
...
+				If you're a member and you’d like to give a Lunch & Learn or Workshop,
+				you can{' '}
+				<a href="https://virtualcoffee.io/lunch-and-learn-idea">
+					submit your idea here
+				</a>
+				.
+			</p>
+			<p>
+				Have a question, suggestion, or want to nominate someone’s act of
+				kindness or blogpost? Email us at{' '}
```
Comment (meg-gutshall) [inline app/routes/__frontend/newsletter/issues/2024-04.jsx:365]: **typo:**

```suggestion
				Have a question, suggestion, or want to nominate someone’s act of
				kindness or blog post? Email us at{' '}
```

**Review by meg-gutshall — COMMENTED** [review]
Hey Ayu! I used [Conventional Comments](https://conventionalcomments.org/) to do my review. There were a few small things that I labeled as nitpicks/non-blocking which you can address or ignore. There are a couple others that should be fixed before this gets posted. Thank you!

File: app/routes/__frontend/newsletter/issues/2024-04.jsx (hunk 2 above)
Comment (adiati98) [reply app/routes/__frontend/newsletter/issues/2024-04.jsx:79]: Credit to whoever write the newsletter! 💖

File: app/routes/__frontend/newsletter/issues/2024-04.jsx (hunk 6 above)
Comment (adiati98) [reply app/routes/__frontend/newsletter/issues/2024-04.jsx:125]: @meg-gutshall we definitely can add it. Can you please let me know the day and time for the check-ins?

**Comment by adiati98** [comment]
Heya @meg-gutshall!

Thanks for the reviews! I've implemented the suggestions.

Just wondering, who's writing the newsletter for email? We might want to inform them to fix the day and time for our events starting from next month. Because I literally copy pasting the content for the website from the email. 😁

Thanks!

File: app/routes/__frontend/newsletter/issues/2024-04.jsx (hunk 6 above)
Comment (meg-gutshall) [reply app/routes/__frontend/newsletter/issues/2024-04.jsx:120]: Can you put down Friday afternoons? We don't have a specific time that we're doing them.

**Comment by meg-gutshall** [comment]
> Just wondering, who's writing the newsletter for email? We might want to inform them to fix the day and time for our events starting from next month. Because I literally copy pasting the content for the website from the email. 😁
> 
> Thanks!

Thanks for letting me know. I'm still learning all the processes here! I think @tkshill does it, but I'm not positive. Let's drop a mention for all the @Virtual-Coffee/maintainers just in case. 😉

**Comment by BekahHW** [comment]
He does write it, but we'll need to update and migrate it to a different service soon. So we'll definitely be making updates.

**Review by meg-gutshall — APPROVED** [review]

**Comment by tkshill** [comment]
@adiati98 I'll make sure the changes are reflected in the next newsletter.

**Pull request merged**